src/public/modules/core/componentViews/avatar-dropdown.html

```diff
@@ -0,0 +1,72 @@
+<div
+  tabindex="0"
+  class="navbar__avatar"
+  ng-class="vm.isDropdownOpen ? 'navbar__avatar--focused' : ''"
+  on-click-outside="vm.isDropdownOpen=false"
+  on-click-watcher="vm.isDropdownOpen"
+  ng-focus="vm.isDropdownOpen=true"
+  ng-mouseenter="vm.isDropdownOpen=true"
+>
+  <button class="navbar__avatar__avatar">
+    {{vm.avatarText}}
+  </button>
+  <div ng-show="!vm.user.contact" class="navbar__avatar__avatar--alert">!</div>
```

Comment:
forgive me if I'm wrong, but isn't this an incorrect usage of BEM? BEM isn't supposed to have nested child elements. an alternative might be to name the parent class `navbar--avatar` (an avatar variation on the navbar), so this becomes `navbar--avatar__avatar--alert`.

Reply:
T_T please ignore it this time, we will be using CSS Modules/ CSS-in-JS moving forward anyways; throwing away BEM!!!